.github/workflows/CI.yml

```diff
@@ -219,7 +220,7 @@ jobs:
         uses: actions/upload-artifact@v2
         with:
           name: msi_artifacts
-          path: build/packages
+          path: $PACKAGING_ROOT
```

Comment:
Does this work? 
>To use the value of an environment variable in a workflow file, you should use the env context. If you want to use the value of an environment variable inside a runner, you can use the runner operating system's normal method for reading environment variables.
>
>If you use the workflow file's run key to read environment variables from within the runner operating system (as shown in the example above), the variable is substituted in the runner operating system after the job is sent to the runner. For other parts of a workflow file, you must use the env context to read environment variables; this is because workflow keys (such as if) require the variable to be substituted during workflow processing before it is sent to the runner.

https://docs.github.com/en/actions/learn-github-actions/environment-variables#about-environment-variables

Reply:
Reasonable to me. That's why some of the actions are not able to use the $VARIABLE convention though. Fixed it with the newest commit.